Wikipedia/Code/WikipediaLookup.swift

```diff
@@ -26,8 +26,30 @@ import CocoaLumberjackSwift
                     localizedName = iOSLocalizedName
                 }
             }
-            return MWKLanguageLink(languageCode: wikipedia.languageCode, pageTitleText: "", name: wikipedia.languageName, localizedName: localizedName)
+            return MWKLanguageLink(languageCode: wikipedia.languageCode, pageTitleText: "", name: wikipedia.languageName, localizedName: localizedName, languageVariantCode: nil)
         }
     }()
-}
 
+    // Flag to be removed once language variants feature can be permanently turned on
+    private static let languageVariantsEnabled = false
+    @objc static let allLanguageVariantsByWikipediaLanguageCode: [String:[MWKLanguageLink]] = {
+        guard languageVariantsEnabled else { return [:] }
+        guard let languagesFileURL = Bundle.wmf.url(forResource: "wikipedia-language-variants", withExtension: "json") else {
+            return [:]
+        }
+        do {
+            let data = try Data(contentsOf: languagesFileURL)
+            let entries =  try JSONDecoder().decode([String : [WikipediaLanguageVariant]].self, from: data)
```

Comment:
Super, super small - but there is an extra space after the `=`, I believe.

Reply:
Done in the new commit.